source/server/options_impl.h

```diff
@@ -200,7 +200,7 @@ class OptionsImpl : public Server::Options, protected Logger::Loggable<Logger::I
   bool hot_restart_disabled_;
   bool signal_handling_enabled_;
   bool mutex_tracing_enabled_;
-  bool core_dump_enabled_;
+  bool core_dump_enabled_ = false;
```

Comment:
Would be good to be consistent on how these fields are initialized in the constructor.  All others are initialized in the explicitly implemented constructor in the cc file.  Consider also initializing this member there or move most of the initialization to this header file instead.

Reply:
Makes sense